### PR TITLE
LF line endings

### LINKED ISSUE #646: Adding windows images to docker manifest
In order to deploy Grafana agent as a windows container running on windows node pools

### PR DESCRIPTION
**Pull request by erikbaranowski**
#### PR Description

LF line endings for web/ui/** in windows

#### Which issue(s) this PR fixes

#646 

#### PR Checklist

- [ ] CHANGELOG updated
- [ ] Documentation added
- [ ] Tests updated

### CODE REVIEW

File: cmd/grafana-agentctl/Dockerfile.windows [hunk 1]
```diff
@@ -7,7 +7,7 @@ WORKDIR /src/agent
 
 SHELL ["cmd", "/S", "/C"]
 
-RUN ""C:\Program Files\git\bin\bash.exe"" -c 'RELEASE_BUILD=${RELEASE_BUILD} VERSION=${VERSION} make agentctl'
+RUN ""C:\Program Files\git\bin\bash.exe" -c 'RELEASE_BUILD=${RELEASE_BUILD} VERSION=${VERSION} make agentctl"
```
Comment (rfratto) [inline cmd/grafana-agentctl/Dockerfile.windows:10]: This doesn't look right, there's a mismatched `'` now. Did you mean 

```
RUN ""C:\Program Files\git\bin\bash.exe" -c 'RELEASE_BUILD=${RELEASE_BUILD} VERSION=${VERSION} make agentctl'"
```

Comment (erikbaranowski) [reply cmd/grafana-agentctl/Dockerfile.windows:10]: so close even though it looks and turns out is wrong. It is the only 1 that runs through make agent just fine but does give a bad return code in the end unfortunately. It just failed locally. Every reasonable quote option is wrong for windows cmd/docker/bash so I'll set it back to the working powershell 1 second

File: cmd/grafana-agent/Dockerfile.windows [hunk 2]
```diff
@@ -5,9 +5,11 @@ ARG RELEASE_BUILD=1
 COPY . /src/agent
 WORKDIR /src/agent
 
-SHELL ["cmd", "/S", "/C"]
+SHELL ["cmd", "/C"]
 
-RUN ""C:\Program Files\git\bin\bash.exe"" -c 'RELEASE_BUILD=${RELEASE_BUILD} VERSION=${VERSION} make generate-ui && make agent'
+RUN "C:\Program Files\git\bin\bash.exe" -c 'RELEASE_BUILD=${RELEASE_BUILD} VERSION=${VERSION} make generate-ui'
+RUN "C:\Program Files\git\bin\bash.exe" -c 'RELEASE_BUILD=${RELEASE_BUILD} VERSION=${VERSION} make agent'
+#RUN powershell "Invoke-Expression ""C:\'Program Files'\git\bin\bash.exe -c 'RELEASE_BUILD=${RELEASE_BUILD} VERSION=${VERSION} make generate-ui && make agent'"""
```
Comment (tpaschalis) [inline cmd/grafana-agent/Dockerfile.windows:12]: Should we remove this comment?

**Comment by tpaschalis** [comment]
Is this solving the issue with the [failing](https://drone.grafana.net/grafana/agent/5372/5/2) Windows-Containerize step?

Looks like the .gitattributes change was removed.

**Comment by rfratto** [comment]
Can you rebase on main to bring in the new CI jobs? And can you also remove [this](https://github.com/grafana/agent/blob/c8b5a46d16e1351ac1f1c82cc8f2acbc16b272b8/.drone/pipelines/check_containers.jsonnet#L48-L50) block after rebasing so building the Windows containers becomes required to succeed?

File: cmd/grafana-agent/Dockerfile.windows (hunk 2 above)
Comment (erikbaranowski) [reply cmd/grafana-agent/Dockerfile.windows]: all set

**Review by rfratto — APPROVED** [review]

**Pull request merged**